site/src/components/Calendar.vue

```diff
@@ -42,6 +42,11 @@
                 backgroundColor: colors(session.section.crn).bg,
                 borderColor: colors(session.section.crn).border,
                 color: colors(session.section.crn).text,
+                scrollbarColor:
+                  colors(session.section.crn).sbThumb +
+                  colors(session.section.crn).sbTrack,
```

Comment:
Do these need a space in between them?

Reply:
firefox seems fine with it ? and I believe firefox is the only browser to support that property.
It might be more proper to have a space there though

Reply:
I'd rather keep a space and/or use a format string to be safe.